package.json

```diff
@@ -25,6 +25,8 @@
     "react-router-dom": "^4.3.1",
     "react-router-hash-link": "^1.2.2",
     "react-scripts": "^3.0.0",
+    "smooth-scroll": "^16.1.0",

```

Comment:
Look good - one suggestion is that we should reduce the number of dependencies. Right now `Home.tsx` implements the same "smooth scroll" functionality using `react-router-hash-link`. 

![Untitled](https://user-images.githubusercontent.com/4195689/65743225-4dc77680-e0b9-11e9-8229-4bf60fa0c699.gif)

Could you use that dependency instead of `smooth-scroll` since `react-router-hash-link` doesn't require you to add additional scripts to the index.html file.

Reply:
Here's the PR w/ code changes for this https://github.com/NuevoFoundation/nuevofoundation-web/pull/156

Reply:
Thanks!! I'll update :)